neutron/plugins/ml2/drivers/openvswitch/agent/ovs_neutron_agent.py

```diff
@@ -183,7 +185,7 @@ def __init__(self, bridge_classes, integ_br, tun_br, local_ip,
         super(OVSNeutronAgent, self).__init__()
         self.conf = conf or cfg.CONF
 
-        self.fullsync = True
+        self.fullsync = False
```

Comment:
That is what was in patch 3 of 7  (along with lots of other changes too).   New logic probably does not want that initialized to True because it needs to finish setting up objects before sync is done.